src/api/comments.ts

```diff
@@ -0,0 +1,32 @@
+import { BASE_URL } from './api';
+
+export function getPostComments(postId: number): Promise<Comment[]> {
+  return fetch(`${BASE_URL}/comments?postId=${postId}`)
+    .then(response => {
+      if (!response.ok) {
+        throw new Error('Something went wrong.');
+      } else {
+        return response.json();
+      }
```

Comment:
else is redundant here
```suggestion
      }
      
      return response.json();
      
```